nativelib/src/main/scala/scala/re2s/UnicodeTables.scala

```diff
@@ -0,0 +1,2483 @@
+  final val Hanunoo                = _Hanunoo
+  final val Hebrew                 = _Hebrew
+  final val Hiragana               = _Hiragana
+  final val Imperial_Aramaic       = _Imperial_Aramaic
+  final val Inherited              = _Inherited
+  final val Inscriptional_Pahlavi  = _Inscriptional_Pahlavi
+  final val Inscriptional_Parthian = _Inscriptional_Parthian
+  final val Javanese               = _Javanese
+  final val Kaithi                 = _Kaithi
+  final val Kannada                = _Kannada
+  final val Katakana               = _Katakana
+  final val Kayah_Li               = _Kayah_Li
+  final val Kharoshthi             = _Kharoshthi
+  final val Khmer                  = _Khmer
+  final val Lao                    = _Lao
+  final val Latin                  = _Latin
+  final val Lepcha                 = _Lepcha
+  final val Limbu                  = _Limbu
+  final val Linear_B               = _Linear_B
+  final val Lisu                   = _Lisu
+  final val Lycian                 = _Lycian
+  final val Lydian                 = _Lydian
+  final val Malayalam              = _Malayalam
+  final val Mandaic                = _Mandaic
+  final val Meetei_Mayek           = _Meetei_Mayek
+  final val Mongolian              = _Mongolian
+  final val Myanmar                = _Myanmar
+  final val New_Tai_Lue            = _New_Tai_Lue
+  final val Nko                    = _Nko
+  final val Ogham                  = _Ogham
+  final val Ol_Chiki               = _Ol_Chiki
+  final val Old_Italic             = _Old_Italic
+  final val Old_Persian            = _Old_Persian
+  final val Old_South_Arabian      = _Old_South_Arabian
+  final val Old_Turkic             = _Old_Turkic
+  final val Oriya                  = _Oriya
+  final val Osmanya                = _Osmanya
+  final val Phags_Pa               = _Phags_Pa
+  final val Phoenician             = _Phoenician
+  final val Rejang                 = _Rejang
+  final val Runic                  = _Runic
+  final val Samaritan              = _Samaritan
+  final val Saurashtra             = _Saurashtra
+  final val Shavian                = _Shavian
+  final val Sinhala                = _Sinhala
+  final val Sundanese              = _Sundanese
+  final val Syloti_Nagri           = _Syloti_Nagri
+  final val Syriac                 = _Syriac
+  final val Tagalog                = _Tagalog
+  final val Tagbanwa               = _Tagbanwa
+  final val Tai_Le                 = _Tai_Le
+  final val Tai_Tham               = _Tai_Tham
+  final val Tai_Viet               = _Tai_Viet
+  final val Tamil                  = _Tamil
+  final val Telugu                 = _Telugu
+  final val Thaana                 = _Thaana
+  final val Thai                   = _Thai
+  final val Tibetan                = _Tibetan
+  final val Tifinagh               = _Tifinagh
+  final val Ugaritic               = _Ugaritic
+  final val Vai                    = _Vai
+  final val Yi                     = _Yi
```

Comment:
These `final val`s which just alias other `final val`s seem to be unnecessary. Why not just make the original public?